FWCore/Framework/interface/BranchActionType.h

```diff
@@ -12,7 +12,8 @@ namespace edm {
     BranchActionGlobalBegin = 0,
     BranchActionStreamBegin = 1,
     BranchActionStreamEnd = 2,
-    BranchActionGlobalEnd = 3
+    BranchActionGlobalEnd = 3,
+    BranchActionGlobalOther = 4
```

Comment:
This name doesn't appear to be very self-explanatory. Would another one be better?

Reply:
So for events which do not have a begin or end we just use `BranchActionStreamBegin`. So for input process block, it is like event (no begin or end) so I suggest using the same convention and just use `BranchActionGlobalBegin`.  Otherwise we should add two new types which deal with the 'instantaneous' nature of event and input process block.

Reply:
I see that `BranchActionGlobalBegin` can't be used for this case as it is already used for 'begin' process block. So instead how about being more descriptive `BranchActionGlobalInput` or very descriptive `BranchActionProcessBlockInput`?

Reply:
I changed the name to BranchActionProcessBlockInput. Thanks, that is a more descriptive name and will help when someone tries to read this code. I'll push a commit soon.